### PR TITLE
Try and clarify minor release weekly before it

### PR DESCRIPTION
**Pull request by danpoltawski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ http://integration.moodle.org jobs chain will start once last version bump is pu
 Spread changes in integration to moodle.git and mirrors using ./release.sh (you may need, on releases, to comment some branches if not releasing all them together).
 
 ### Minor release
+**Note:** Minor releases are usually produced on the weekend, a normal weekly is produced for stable supported branches on the weekdays before it.
```
Comment (stronk7) [inline README.md]: normal weekly => \*\*normal weekly\*\* ?

Comment (stronk7) [inline README.md]: And what about adding something like, to the weeklies section:

    **2. Review the branches very carefully.**

    Note that weekly rolls involve, exclusively, supported branches. Always. No distinction the weeks before releases or the on-demand/on-sync explained below.

    Only exception is when, after agreement, it has been decided to force something to land into unsupported branches and configuration is altered to cause that.

Comment (danpoltawski) [inline README.md]: done

**Comment by stronk7** [comment]
Merged thanks!

**Pull request merged**